packages/react-core/src/components/ClipboardCopy/ClipboardCopy.tsx

```diff
@@ -86,7 +90,7 @@ export class ClipboardCopy extends React.Component<ClipboardCopyProps, Clipboard
     isReadOnly: false,
     isExpanded: false,
     isCode: false,
-    variant: 'inline',
+    variant: null,
```

Comment:
We can't change the API like this.  We are redefining what we were calling inline before.  this could break consumers that where already using the "inline" variation.  we will have to call the inline variation something else. We can refactor when we have a breaking change release...
cc @mcoker @mcarrano

Reply:
I'll add a variant value `inline-compact` for the new inline styling for the interim. We can update properly (`inline`, `expansion` and maybe `block` for the variant values) when we have a breaking change release.